test/mpi/configure.ac

```diff
@@ -371,6 +372,9 @@ if test -n "$with_mpi" ; then
     else
         CXX=$MPICXX
     fi
+    if test -z "$MPIEXEC" ; then
+        MPIEXEC=$with_mpi/bin/MPIEXEC_NAME
```

Comment:
should be $MPIEXEC_NAME

Reply:
I'm still going to approve assuming that bit gets fixed. It worked when I changed it locally.

Reply:
Fixed. Will merge.